helm-chart/eoapi/tests/config_tests.yaml

```diff
@@ -44,3 +47,18 @@ tests:
       - equal:
           path: data.WEB_CONCURRENCY
           value: "5"
+  - it: "multidim configmap defaults"
+    set:
+      raster.enabled: false
+      stac.enabled: false
+      vector.enabled: false
+      multidim.enabled: true
+    asserts:
+      - isKind:
+          of: ConfigMap
+      - matchRegex:
+          path: metadata.name
+          pattern: ^multidim-envvar-configmap-RELEASE-NAME$
+      - equal:
+          path: data.GDAL_HTTP_MULTIPLEX
+          value: "YES"
```

Comment:
Guess checking one implies the rest worked? Or is this value *very* special to check?